docs/themes/avocadocs/layouts/shortcodes/repolist.html

```diff
@@ -10,25 +10,25 @@
 
     <div class="card">
       <div class="card-header-template" id="heading{{ .shortname }}">
-        <h5 class="template-card">
+        <div class="template-card">
```

Comment:
3 divs inside each other seems like a code smell.  Shouldn't some of them be section or article or aside or something with more semantic meaning?  Do we even need all 3?  Seems sketchy.

Reply:
I can change this back. This is just left over from testing the split.

Reply:
Let's change it back for now at least.  We can look into a markup cleanup later.

Reply:
Forgot to resolve, this is back to an h5 now